subgraph/schema.graphql

```diff
@@ -52,6 +52,8 @@ type Key @entity {
   createdAtBlock: BigInt!
   "Invoked by a Lock manager to expire the user's key and perform a refund and cancellation of the key"
   cancelled: Boolean
+  "hash of the transaction"
+  transactionsHash: [String!]
```

Comment:
So this will store all transactions related to this specific tokenId ? Better to explicit a bit better in the comment above maybe

Reply:
added better comment also where `transactionsHash` are set